routes/web.php

```diff
@@ -16,3 +16,6 @@
 Route::get('/', function () {
     // return your computed table here
 });
+
+
+Route::get('/products',"App\Http\Controllers\ProductsController@list");
```

Comment:
Usando la sintassi piú recente
```suggestion
Route::get('/products',[App\Http\Controllers\ProductsController::class, "list"]);
```